.buildkite/ftr_configs.yml

```diff
@@ -274,3 +274,4 @@ enabled:
   - x-pack/performance/journeys/promotion_tracking_dashboard.ts
   - x-pack/performance/journeys/web_logs_dashboard.ts
   - x-pack/performance/journeys/data_stress_test_lens.ts
+  - x-pack/test/banners_functional/config.ts
```

Comment:
I already added this suite back in my previous PR (check line 129). We need to remove either one of them.

Reply:
Ah that makes sense.  whoops!